src/pyTRLCConverter/rst_converter.py

```diff
@@ -0,0 +1,701 @@
+        """
+        Return converter description.
+
+        Returns:
+            str: Converter description
+        """
+        return "Convert into reStructuredText format."
+
+    @classmethod
+    def register(cls, args_parser: any) -> None:
+        # lobster-trace: SwRequirements.sw_req_rst_multiple_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_single_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_default
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_custom
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_default
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_custom
+        """
+        Register converter specific argument parser.
+
+        Args:
+            args_parser (any): Argument parser
+        """
+        super().register(args_parser)
+
+        BaseConverter._parser.add_argument(
+            "-e",
+            "--empty",
+            type=str,
+            default=BaseConverter.EMPTY_ATTRIBUTE_DEFAULT,
+            required=False,
+            help="Every attribute value which is empty will output the string." \
+                f"(default = {BaseConverter.EMPTY_ATTRIBUTE_DEFAULT})."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-n",
+            "--name",
+            type=str,
+            default=RstConverter.OUTPUT_FILE_NAME_DEFAULT,
+            required=False,
+            help="Name of the generated output file inside the output folder " \
+                f"(default = {RstConverter.OUTPUT_FILE_NAME_DEFAULT}) in " \
+                "case a single document is generated."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-sd",
+            "--single-document",
+            action="store_true",
+            required=False,
+            default=False,
+            help="Generate a single document instead of multiple files. The default is to generate multiple files."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-tl",
+            "--top-level",
+            type=str,
+            default=RstConverter.TOP_LEVEL_DEFAULT,
+            required=False,
+            help="Name of the top level heading, required in single document mode. " \
+                f"(default = {RstConverter.TOP_LEVEL_DEFAULT})"
+        )
+
+    def begin(self) -> Ret:
+        # lobster-trace: SwRequirements.sw_req_rst_single_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level
+        """
+        Begin the conversion process.
+
+        Returns:
+            Ret: Status
+        """
+        assert self._fd is None
+
+        result = Ret.OK
+
+        # Single document mode?
+        if self._args.single_document is True:
+            log_verbose("Single document mode.")
+        else:
+            log_verbose("Multiple document mode.")
+
+        # Set the value for empty attributes.
+        self._empty_attribute_value = self._args.empty
```

Comment:
Not sure if this aliasing of _args values is useful if the value isn't changed at runtime.

The code uses both direct access to self._args.X and  self.Y = self._args.Y conventions without a clear distinction.

Recommendation: Don't  alias unless the value needs modifying.
(Note: Same for _args.out.)

Reply:
Its overrides the base converter default.